style.css

```diff
@@ -1,106 +1,107 @@
-*{
-    margin:0;
-    padding:0;
-    box-sizing:border-box;
-    font-family: cursive, sans-serif;
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: cursive, sans-serif;
 }
-.left-sidebar{
-    position: absolute;
-    width: 245px;
-    height: 80px;
-    left: 0px;
-    top: 0px;
-    background: #FFFFFF;  
+
+.left-sidebar {
+  position: absolute;
+  width: 245px;
+  height: 80px;
+  left: 0px;
+  top: 0px;
+  background: #ffffff;
 }
-.header{
-    position: absolute;
-    width: 1127px;
-    height: 82px;
-    left: 239px;
-    top: -2px;
-    background: #FFFFFF;
-    border: 2px solid #EAEDF3;
+.header {
+  position: absolute;
+  width: 1127px;
+  height: 82px;
+  left: 239px;
+  top: -2px;
+  background: #ffffff;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#ffffff" to be "#fff" (color-hex-length)](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5476667)